src/test/java/teammates/test/cases/webapi/CreateInstructorActionTest.java

```diff
@@ -47,27 +48,18 @@ protected void testExecute() throws Exception {
 
         String[] submissionParams = new String[] {
                 Const.ParamsNames.COURSE_ID, courseId,
-                Const.ParamsNames.INSTRUCTOR_NAME, newInstructorName + " ", // with some extra spaces
-                Const.ParamsNames.INSTRUCTOR_EMAIL, " " + newInstructorEmail + " ", // with some extra spaces
-
-                Const.ParamsNames.INSTRUCTOR_ROLE_NAME,
-                Const.InstructorPermissionRoleNames.INSTRUCTOR_PERMISSION_ROLE_COOWNER,
+        };
 
-                Const.ParamsNames.INSTRUCTOR_DISPLAY_NAME,
+        InstructorCreateRequest reqBody = new InstructorCreateRequest(instructorId, newInstructorName, newInstructorEmail,
                 Const.InstructorPermissionRoleNames.INSTRUCTOR_PERMISSION_ROLE_COOWNER,
+                null, false);
 
-                Const.ParamsNames.INSTRUCTOR_PERMISSION_MODIFY_COURSE, "true",
-                Const.ParamsNames.INSTRUCTOR_PERMISSION_MODIFY_INSTRUCTOR, "true",
-                Const.ParamsNames.INSTRUCTOR_PERMISSION_MODIFY_SESSION, "true",
-                Const.ParamsNames.INSTRUCTOR_PERMISSION_MODIFY_STUDENT, "true",
-        };
+        CreateInstructorAction createInstructorAction = getAction(reqBody, submissionParams);
+        JsonResult actionOutput = getJsonResult(createInstructorAction);
 
-        CreateInstructorInCourseAction a = getAction(submissionParams);
-        JsonResult r = getJsonResult(a);
+        assertEquals(HttpStatus.SC_OK, actionOutput.getStatusCode());
 
-        assertEquals(HttpStatus.SC_OK, r.getStatusCode());
-
-        MessageOutput msg = (MessageOutput) r.getOutput();
+        MessageOutput msg = (MessageOutput) actionOutput.getOutput();
         assertEquals("The instructor " + newInstructorName + " has been added successfully. "
```

Comment:
Fix this to use `COURSE_INSTRUCTOR_ADDED` from `Const.java`.

Reply:
Done 👍